front_end/src/app/admin/events/family-finder/household/household.component.html

```diff
@@ -0,0 +1,18 @@
+<header-event></header-event>
+
+<div class="container-fluid">
+  <preloader *ngIf="processing"></preloader>
+  <div class="mt-2 clearfix" *ngIf="eventParticipants && !processing">
```

Comment:
Could this be consolidated into a function that can be tested in the component spec? We have two conditions performing some logic.